packages/api/src/routes/medical/internal-docs.ts

```diff
@@ -346,12 +346,12 @@ router.post(
 
     const fileData = documentDataSchema.parse(req.body);
 
-    await updateDocumentReference({
+    const docRef = await updateDocumentReference({
       cxId,
       fileData,
     });
 
-    return res.sendStatus(httpStatus.OK);
+    return res.status(httpStatus.OK).json(docRef);
```

Comment:
nit: isn't this function called by a lambda? so we don't need to see the json return? unless i missed another place it's used in

Reply:
Yah so the document-uploader calls this endpoint to update the docRef, and it needs the docRef itself in order to generate the metadata file that we store in s3, and we use for responding to DQs.

Reply:
So returning the docRef here is needed

Reply:
Ah, gotcha 👍